### DISCUSSION TITLE
Can't update specific table on Postgres

### DISCUSSION THREAD

**Discussion by delucca** [directus/directus#7932] [Archive]
Hi everyone.

I've configured my project with Directus, and it is running smoothly. The only issue I'm facing is specific with my "User" table.

I can update the data, remove and create new entities on any other table, but if I try to update my "User" table Directus simply doesn't work. By that I mean, it keeps loading without any other response or error.

I've already checked my container logs, but it throws me no error.

My user table is pretty simple, it contains the following columns:
- id `uuid` `UNIQUE`
- first_name `string`
- last_name `string`
- email `string` `UNIQUE`
- nickname `string`
- picture `string`
- gender `enum`
- role `string`
- linked_in_profile_address `string`
- about `string`
- authz_sub `string`
- created_at `date`
- updated_at `date`

Also, the table name is "user". In Postgres we always need to use double-quotes in those cases, since using `user` directly would fetch data from roles.

But, this issue is quite strange, before I model my data I was able to update a given user e-mail directly into Directus.

What should I do to troubleshoot this?

**Comment by rijkvanzanten** [comment]
Would you be able to share a database dump we can use to reproduce this table locally?

**Reply by delucca** [reply]
Sure! Here it is: https://drive.google.com/file/d/1hVd2sIZNJ5NLytLqCaSbwuz1baFQodGJ/view?usp=sharing

**Reply by delucca** [reply]
Hi @rijkvanzanten, sorry to bother you, were you able to take a look on this?

**Reply by rijkvanzanten** [reply]
@delucca I tried taking a look, but the file is no longer available. Can you share a copy here?

**Reply by delucca** [reply]
Sorry about that, I've removed the file. Now it is available again

**Reply by rijkvanzanten** [reply]
Lets continue this convo on https://github.com/directus/directus/issues/8047#issuecomment-924967543 before we go cross-eyed 😄
